bigchaindb/backend/query.py

```diff
@@ -5,6 +5,7 @@
 from bigchaindb.backend.exceptions import OperationError
 
 VALIDATOR_UPDATE_ID = 'a_unique_id_string'
+PRE_COMMIT_ID = 'a_unique_id_string'
```

Comment:
*No need to address this comment in this PR.*

I have the feeling those unique ids are necessary because we have two different collections for two specific use cases, one is to keep track of the last validator to update, the other one is for storing the pre commit id. I know it's outside the scope of this PR, but an idea is to have (in the future) a collection to keep the current state of the application, something like `state`. In `state` we have two documents, one for the validator to update, the other one for the pre commit id. The first one will have key `validator_update`, the second one will have key `pre_commit`.